controller.php

```diff
@@ -14,8 +14,8 @@ class Controller extends Package
 {
 
 	protected $pkgHandle = 'theme_majorca';
-	protected $appVersionRequired = '8.0.0';
-	protected $pkgVersion = '1.0.2';
+	protected $appVersionRequired = '9.0.0';
+	protected $pkgVersion = '2.0.2';
```

Comment:
@exa18 

Thank you for your contribution! I'll test it, but the changes in this pull request do not require a minimum supported version change. Can you keep it, and change the package version to 1.1.0?